frontend/src/concepts/projects/ComponentWithProjectLink.tsx

```diff
@@ -0,0 +1,27 @@
+import { Divider, Flex } from '@patternfly/react-core';
+import React from 'react';
+import ProjectNavigatorLink from '~/concepts/projects/ProjectNavigatorLink';
+import { ProjectKind } from '~/k8sTypes';
+
+type ComponentWithProjectLinkProps = {
+  children: React.ReactNode;
+  project: ProjectKind;
+};
+
+const ComponentWithProjectLink: React.FC<ComponentWithProjectLinkProps> = ({
+  project,
+  children,
+}) => (
+  <Flex spaceItems={{ default: 'spaceItemsSm' }}>
+    {children}
+    <Divider
+      orientation={{
+        default: 'vertical',
+      }}
+      style={{ height: '1rem', alignSelf: 'center', marginRight: 0 }}
```

Comment:
are these custom styles needed

Reply:
you should be able to do this
<img width="438" alt="Screenshot 2025-05-30 at 2 03 41 PM" src="https://github.com/user-attachments/assets/429c012f-3540-4b4e-a5bd-d82cb9ab0f49" />

```tsx
<Breadcrumb ouiaId="BasicBreadcrumb">
    <BreadcrumbItem to="#">Section home</BreadcrumbItem>
    <BreadcrumbItem to="#">Section title</BreadcrumbItem>
    <BreadcrumbItem to="#">Section title</BreadcrumbItem>
    <Flex>
      <Divider orientation={{default: "vertical"}}/>
      <FlexItem>
        <Button isInline variant="link">I am a button</Button>
      </FlexItem>
    </Flex>
  </Breadcrumb>
```

Reply:
you gotta use Content small

```tsx
<Breadcrumb ouiaId="BasicBreadcrumb">
    <BreadcrumbItem to="#">Section home</BreadcrumbItem>
    <BreadcrumbItem to="#">Section title</BreadcrumbItem>
    <BreadcrumbItem to="#">Section title</BreadcrumbItem>
    <Flex>
      <Divider orientation={{default: "vertical"}}/>
      <FlexItem>
      <Content component={ContentVariants.small}>
        <Button isInline variant="link">I am a button</Button>
        </Content>
      </FlexItem>
    </Flex>
  </Breadcrumb>
```

as described in the other comment